src/javascript/app/pages/user/account/change_password.js

```diff
@@ -1,45 +1,259 @@
+
+        // Handle unlinking social signup
+        $(social_unlink_btn_id).off('click').on('click', () => {
+            Dialog.confirm({
+                id               : 'unlink_confirmation_dialog',
+                localized_message: localize('You will need to set a password to complete the process.'),
+                localized_title  : localize('Are you sure you want to unlink from [_1]?', social_signup_identifier),
+                cancel_text      : localize('Cancel'),
+                ok_text          : localize('Unlink'),
+                onConfirm        : () => onSentEmail('unlink'),
+            });
+        });
+    };
+
+    const initTradingPassword = () => {
+        $trading_password_container.removeClass('invisible');
+        $set_trading_password_container.addClass('invisible');
+
+        // Handle change trading password
+        FormManager.init(trading_form_id, [
+            { selector: '#old_trading_password', request_field: 'old_password', validations: ['req', ['length', { min: 6, max: 25 }]], clear_form_error_on_input: true },
+            { selector: '#new_trading_password', request_field: 'new_password', validations: ['req', 'password', ['not_equal', { to: '#old_trading_password', name1: localize('Current password'), name2: localize('New password') }], 'compare_to_email'] },
+            { request_field: 'trading_platform_password_change', value: 1 },
+        ]);
+        FormManager.handleSubmit({
+            form_selector       : trading_form_id,
+            fnc_response_handler: tradingPwHandler,
+            enable_button       : true,
+        });
+
+        // Handle forgot trading password
+        $(forgot_trading_pw_btn_id).off('click').on('click', () => {
+            onSentEmail('trading_password');
+        });
+    };
+
+    const initSetTradingPassword = () => {
+        $set_trading_password_container.removeClass('invisible');
+
+        // Handle set trading password
+        FormManager.init(trading_form_id, [
+            { selector: '#set_new_trading_password', request_field: 'new_password', validations: ['req', 'password', 'compare_to_email'] },
+            { request_field: 'old_password', value: '' },
+            { request_field: 'trading_platform_password_change', value: 1 },
+        ], null, shouldSetTradingPassword());
+        FormManager.handleSubmit({
+            form_selector       : trading_form_id,
+            fnc_response_handler: setTradingPwHandler,
         });
     };
 
-    const handler = (response) => {
+    const changePasswordHandler = (response) => {
         if ('error' in response) {
-            $(`${form_id}_error`).text(response.error.message).setVisibility(1);
+            const frm_change_binary_password_error_id = '#frm_change_binary_password_error';
+            $(frm_change_binary_password_error_id).text(response.error.message).setVisibility(1);
+            setTimeout(() => {
+                $(frm_change_binary_password_error_id).setVisibility(0);
```

Comment:
```suggestion
            const $_frm_change_binary_password_error_id = $('#frm_change_binary_password_error');
            $_frm_change_binary_password_error_id.text(response.error.message).setVisibility(1);
            setTimeout(() => {
                $_frm_change_binary_password_error_id.setVisibility(0);
```

Reply:
couldn't find any case of using `$_` prefixed to the variable name, I will proceed with prefixing the name by just  a `$`